test/request.deprecated.test.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+// Tests for some deprecated `request.*` keys. This file should be
+// removed when the deprecation is complete.
+
```

Comment:
The scope of isolating the deprecated getter into a test is to suppress the warning when you run `npm test` locally.

So we need to fix this one too:

> test/route.test.js 2> (node:72195) [FSTDEP012] FastifyDeprecation: Request#context
property access is deprecated. Please use "Request#routeConfig" or
"Request#routeSchema" instead for accessing Route settings. The "Request#context"
will be removed in `fastify@5`.

This suggestion fixes the warning from this file

```suggestion

process.removeAllListeners('warning')

```